pkg/modules/webhook/middleware_test.go

```diff
@@ -387,6 +388,27 @@ func TestWebhookMiddlewareAsynchronousProcess(t *testing.T) {
 			expectWebhookFilename:         "foo",
 			expectWebhookExtraHTTPHeaders: map[string]string{"foo": "bar"},
 		},
+		{
+			request: func() *http.Request {
+				req := buildMultipartFormDataRequest()
+				req.Header.Set("Gotenberg-Output-Filename", "foo")
+
+				return req
+			}(),
+			mod: buildWebhookModule(),
+			next: func() echo.HandlerFunc {
+				return func(c echo.Context) error {
+					ctx := c.Get("context").(*api.Context)
+
+					return ctx.AddOutputPaths("/tests/test/testdata/api/sample3.pdf")
+				}
+			}(),
+			returnedError: echo.ErrInternalServerError,
```

Comment:
> retryablehttp performs automatic retries under certain conditions. Mainly, if an error is returned by the client (connection errors, etc.), or if a 500-range response code is received (except 501), then a retry is invoked after a wait period. Otherwise, the response is returned and left to the caller to interpret.

Testing a `5xx` will trigger a retry, and fails with a standard error. You should test a `Bad Request` IMO.

Reply:
Ah right, I thought it then retries for n times and returns the final error, I've changed the test to a `Bad Request`.